paper.md

```diff
@@ -26,90 +26,69 @@ affiliations:
 
-\begin{multicols}{2}
-\begin{figure}
-\centering
-\includegraphics[width=0.45\textwidth,height=0.45\textheight]{paper_img/single_source_shortest_path.png}
-\caption{Time to find the shortest path between two nodes in the USA road network.}
-\label{fig:sssp}
-\end{figure}
-
-\begin{figure}
-\centering
-\includegraphics[width=0.45\textwidth,height=0.45\textheight]{paper_img/all_pairs.png}
-\caption{Time to find the shortest path among all nodes in the City of Rome road network.}
-\label{fig:allpairs}
-\end{figure}
-\end{multicols}
-
-_retworkx_ is 6x faster than _NetworkX_ on the single-source scenario, and 104x faster on the all-pairs scenario as shown in Figures \ref{fig:sssp} and \ref{fig:allpairs}. We highlight that _NetworkX_ is the slowest library among all in the benchmark because it was designed with different goals in mind, such as readability and ease of distribution.
-
-_retworkx_ is the second-fastest in the single-source scenario after _graph-tool_, but we interpret the result as a trade-off. _graph-tool_ is 1.6-1.8x faster than _retworkx_ on the single-source shortest-path calculation but takes 5x longer to create the graph, as shown in the other benchmark. _retworkx_ creates graphs faster and trades some of the shortest-path calculation performance to accept Python callbacks that work with arbitrary data types. In the all-pairs scenario, _retworkx_ is the fastest with a 5.6x speedup compared to _graph-tool_ which is the second-fastest.
+In addition, Qiskit needs to traverse the graph. Some transformations, such as greedily merging instructions to reduce circuit depth, require graph traversal. _retworkx_ offers the methods for traversals such as breadth-first search, depth-first search, and topological sorting.
 
 ## Subgraph Isomorphism
 
-The third use case is to detect a pattern graph within a larger graph using the VF2 or VF2++ algorithms [@Cordella2004;@Juttner2018]. We compare the time to answer if pairs of graphs from the ARG Database are subgraph-isomorphic [@DeSanto2003]. The graphs are unlabeled, bounded-valence graphs ranging from $20$ to $1000$ nodes with valence $\upsilon \in \{3, 6, 9 \}$. They are organized in pairs such that the subgraph size is either $20 \%$, $40 \%$ or $60 \%$ of the full graph.
+The second use case is based on qubit mapping problem for Noisy Intermediate-Scale Quantum (NISQ) devices [@Bharti2021]. NISQ devices do not have full connectivity among qubits, hence Qiskit needs to take into account an undirected graph representing the connectivity of the device when compiling quantum circuits. Qiskit transforms the quantum circuit such that the pairs of qubits executing two-qubit gates respect the device's architectural constraints. There are many proposed solutions to the qubit mapping problem, including algorithms based on subgraph isomorphism [@Li2021].
```

Comment:
I think that the reference here, should be the original NISQ paper written by Preskill, [reference](https://quantum-journal.org/papers/q-2018-08-06-79/).

Reply:
I cited both papers, the Bharti et al. paper has more details about qubit mapping so I kept it